packages/strapi-plugin-content-manager/admin/src/index.js

```diff
@@ -32,6 +32,7 @@ export default strapi => {
         key: 'content-manager.link',
       },
     ],
+    injectionZones: { listView: { actions: [] } },
```

Comment:
Yes it's necessary. Otherwise, the InjectionZone can't use the `getInjectedComponents` (in the actual situation)

However, it looks technically feasible to me to remove that key  and let the injection zones check what has been registered in the boot function (and it would reduce the noise in the config 🚀 )